_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -23,28 +23,15 @@ The rule applies to any [keyboard event][]:
  - the method `getModifierState` returns `false`; and
  - is [dispatched][] to an [event target][] within a [HTML document][].
 
-## Expectation 1
+## Expectation
 
 For each test target at least one of the following is true:
- - (**Disable**:) a set of one or more [instruments][instrument] are available to prevent any result of the [keyboard event][] within a 10 minute time span of the [event firing][]; or
- - (**Focus**:) any result of the event is prevented if the [event target][] doesn't have [focus][] within a 10 minute time span of the [event firing][]; or
- - (**Remap**:) a set of one or more [instruments][instrument] are available to prevent any result of the [keyboard event][] when the [keyboard event][] method `getModifierState` returns `false` within a 10 minute time span of the [event firing][].
-
-**Note:** The same [instrument][] can be used to **disable**/**remap** more than one event.
-
-**Note:** Preventing results of the event can be done in multiple ways (e.g. removing the event listener; handling the event in a different manner; ...) but the way in which it is done is not relevant for this rule.
+ - (**Disable**:) There is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] of the registered event or can be found in a [clearly labeled location][] from that [web page][]. The set of [instruments][instrument] can be used to prevent [changes to the content][changes in content] of the [web page][] resulting from the [keyboard event][] within a 10 minute time span of the [event firing][]; or
+ - (**Focus**:) If the [event target][] doesn't have [focus][] there are no [changes to the content][changes in content] resulting from the [keyboard event][] within a 10 minute time span of the [event firing][]; or
+ - (**Remap**:) There is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] of the registered event or can be found in a [clearly labeled location][] from that [web page][]. The set of [instruments][instrument] can be used to prevent [changes to the content][changes in content] of the [web page][] within a 10 minute time span of the [event firing][] when the [keyboard event][] method `getModifierState` returns `false`;
```

Comment:
And looking at this now that it streamlined, I'm wondering if (**Disable**) is really needed…

Everything that pass *Disable** will also pass **Remap**: if the event is always blocked, then it is blocked when `getModifierState` is false…

We could have **Remap** be refined ("…and the event isn't blocked when `getModifierState` is true"). But I am actually not sure it is needed…

With a note (or rather example description, we avoid notes now…), it could be pointed out that disabling the event completely does indeed pass the rule. **Remap** would have to be renamed, though…